From: Robert Luberda <robert@debian.org>
Date: Sun, 20 Mar 2022 23:04:39 +0100
Subject: Use bash in sa2 script

Fix the following lintian message:
I: sysstat: bash-term-in-posix-shell '${RANDOM}' [usr/libexec/sysstat/sa2:30]
---
 sa2.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sa2.in b/sa2.in
index 535be72..69b59b2 100644
--- a/sa2.in
+++ b/sa2.in
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 # @SA_LIB_DIR@/sa2
 # (C) 1999-2023 Sebastien Godard (sysstat <at> orange.fr)
 #
